perception/autoware_image_projection_based_fusion/config/pointpainting.param.yaml

```diff
@@ -9,7 +9,6 @@
     post_process_params:
       # post-process params
       circle_nms_dist_threshold: 0.3
-      iou_nms_target_class_names: ["CAR"]
```

Comment:
Why are we removing this parameter?

(Also, if we change this, don't we need to update "Interface changes" part?)

Reply:
> Interface changes
I mentioned this change. Thanks for remind.

The parameter seems to be implemented only for those classes, but the implementation didn't limit the NMS process
As of today, the NMS should be implemented all of the classes. Only exemption is between pedestrian and non-pedestrian.

Reply:
Ah true, now make sense to me, thanks for the explanation 🙏